docs/installation.md

```diff
@@ -21,45 +14,51 @@ If you are using [`ghcup`](https://www.haskell.org/ghcup/) to manage your instal
 ghcup install hls
 ```
 
-You can check if HLS is available for your platorm via ghcup here: <https://haskell.org/ghcup/install/#supported-platforms>.
+You can check if HLS is available for your platform via `ghcup` here: <https://haskell.org/ghcup/install/#supported-platforms>.
 
 You can also install HLS from source without checking out the code manually:
 
 ```bash
-ghcup compile hls -v 1.6.1.0 --ghc 8.10.7
+# `ghcup compile` uses cabal under the hood to build, so you may want to run `cabal update` beforehand
+cabal update
+ghcup compile hls -v $HLS_VERSION --ghc $GHC_VERSION
 ```
 
-Install HLS for multiple GHC versions:
-```
-ghcup compile hls -v 1.6.1.0 --ghc 8.10.7 --ghc 8.8.4
+For example:
+
+Install HLS 1.7.0.0 for GHC 8.10.7, or for multiple GHC versions:
+```bash
+ghcup compile hls -v 1.7.0.0 --ghc 8.10.7
+ghcup compile hls -v 1.7.0.0 --ghc 8.10.7 --ghc 9.2.1
 ```
 
-Use a different `cabal.project` for a GHC version:
+Install HLS from master, or a specific commit:
```

Comment:
I wonder if we should explain the caveat here that this may overwrite existing binary if the hls version on master happens to be 1.7.0.0 and whether we should explain `--overwrite-version` switch.

Reply:
I feel like I'm writing the docs for `ghcup compile` in the HLS docs :) I don't want to explain every in and out of how the command works, partly because I don't know! I think referring people to the `--help` output (which we do) has to be good enough, unless you do actually end up adding something to the ghcup user guide about it.